docs/versioned_docs/version-2.11.0/deprecated/ugcstor.md

```diff
@@ -0,0 +1,2496 @@
+```
+
+The successful deletion of a cStor volume can be verified by running the following commands and ensure there is no entries of particular volume exists as part of the output.
+
+Verify the PVC is deleted successfully using the following command.
+
+```
+kubectl get pvc -n <namespace>
+```
+
+Verify the PV is deleted successfully using the following command:
+
+```
+kubectl get pv
+```
+
+Verify if the cStorVolume is deleted successfully using the following command:
+
+```
+kubectl get cstorvolume -n <openebs_installed_namespace>
+```
+
+Verify if the cStorVolumeReplica(CVR) is deleted successfully using the following command:
+
+```
+kubectl get cvr -n <openebs_installed_namespace>
+```
+
+Verify corresponding cStor Volume target also deleted successfully using the following command:
+
+```
+kubectl get pod -n <openebs_installed_namespace> | grep <pvc_name>
+```
+
+### Patching pool deployment by adding or modifying resource limit and requests
+
+1. Create a patch file called "patch.yaml" and add the following content to it. You can change the values based on the Node configuration. Recommended values are 4Gi for limits and 2Gi for requests.
+
+   ```
+   spec:
+     template:
+       spec:
+         containers:
+         - name: cstor-pool
+           resources:
+             limits:
+               memory: 4Gi
+             requests:
+               memory: 2Gi
+   ```
+
+2. Get the pool deployment using the following command:
+  
+   ```
+   kubectl get deploy -n openebs    
+   ```
+  
+3. Patch the corresponding pool deployment using the following command.
+
+   ```
+   kubectl patch deployment <pool_deployment_name> --patch "$(cat patch.yaml)" -n <openebs_installed_namespace>
+   ```
+
+   Eg: 
+
+   ```
+   kubectl patch deployment <pool_deployment_name> --patch "$(cat patch.yaml)" -n openebs
+   ```
+
+   **Note:** After patching, the existing pool pod will be terminated and a new pool pod will be created. Repeat the same process for other deployments of the same pool as well one by one once new pool pod is created.
+
+## Admin Operations
+
+### Creating cStor Storage Pools
+
+The cStorStoragePool can be created by specifying the blockDeviceList. The following section will describe the steps in detail. 
+
+#### Create a cStorPool by specifying blockDeviceList
+
+**Overview**
+
+1. Get the details of blockdevices attached in the cluster.
+2. Create a StoragePoolClaim configuration YAML and update the required details.
+3. Apply the StoragePoolClaim configuration YAML to create the cStorStoragePool.
+
+**Step1:**
+
+Get all the blockdevices attached in the cluster with the following command. Modify the following command  with appropriate namespace where the OpenEBS is installed. The default namespace where OpenEBS is getting installed is `openebs`.
+
+```
+kubectl get blockdevice -n <openebs_namespace>
+```
+
+Example:
+
+```
+kubectl get blockdevice -n openebs
+```
+
+The output will be similar to the following.
+
+```
```

Comment:
The output section dont require the copy button, in docs.openebs.io we used to remove the button we were using css styled block. Here  we can use `hideCopy`.

sample: `` ```<languagename> hideCopy``` ``
ex: `` ```shell hideCopy``` ``
language name is important, you can give anything, its just for taking the hideCopy as param

Reply:
![image](https://user-images.githubusercontent.com/75615087/128318971-ef6f3d25-332d-4b36-89da-af74d3efc298.png)
screenshot from old doc